### PR TITLE
docs: fixes for japanese translations

### PR DESCRIPTION
**Pull request by emmambd**
This PR 

- Applies suggested translations fixes for Japanese content
- Updates homepage and Training content on the GTFS Schedule section to match with English version

### CODE REVIEW

File: docs/realtime/index.ja.md [hunk 1]
```diff
@@ -14,31 +14,33 @@ search:
...
+*   ルート更新情報 - 遅延、中止、ルートの変更
+*   運行情報 - 停車地の変更や、駅、ルート、またはネットワーク全体に影響する不測の事態
+*   車両位置 - 位置と混雑度を含む車両に関する情報
 
-GTFS Realtimeは、車両の_実際の_状態を表示することができるため、フィードは定期的に更新する必要があります - できれば、新しいデータは自動車両位置情報システムから入ってくることが望ましいです。
+フィードは異なるタイプのエンティティを組み合わせることもできますが、必須ではありません。 フィードは HTTP 経由で提供され、更新頻度は高いものとなります。ファイル自体は通常のバイナリ ファイルなので、どの種類のウェブサーバーでもファイルのホストと配信することができます（他の転送プロトコルも使用できます）。 また、Web アプリケーション サーバーを使用した場合は、有効な HTTP GET リクエストに対するレスポンスとしてフィードを返すこともできます。 フィードの更新頻度や取得方法については、特に制約はありません。
+
+GTFS リアルタイムは、車両の「現在の」状態を表示することができるため、フィードは定期的に更新する必要があります。 できれば自動車両位置特定システムから新しいデータが受信されるたびに更新されることが望ましいです。
 
 [フィードエンティティについて](feed-entities)
```
Comment (tzujenchanmbd) [inline docs/realtime/index.ja.md:46]: Line 47 - 53 "データ形式" need update

File: docs/realtime/changes/index.ja.md [hunk 2]
```diff
@@ -34,14 +34,14 @@ GTFS Realtime に貢献するには、[仕様変更](../process)プロセスを
 </div>
 <div class="row"></div> -->
 
-## 最近マージされたプロポーザル &ensp;<img src="../../assets/pr-merged.svg" style="height:1em;"/>
+## 最近追加された提案 &ensp;<img src="../../assets/pr-merged.svg" style="height:1em;"/>
 
-最近マージされたプロポーザルは、[公式のGTFS Realtime 仕様](../reference)として機能するようになりました。詳しくは[リビジョンヒストリーを](../process#revision-history)ご覧ください。
+最近追加された提案は、[公式のGTFS Realtime 仕様](../reference)として機能するようになりました。詳しくは[リビジョンヒストリーを](../process#revision-history)ご覧ください。
```
Comment (tzujenchanmbd) [inline docs/realtime/changes/index.ja.md]: exclude "を" from link

**Review by tzujenchanmbd — CHANGES_REQUESTED** [review]
[ja_realtime] 

- Missing 1 section
- revise 1 link

File: docs/realtime/index.ja.md (hunk 1 above)
Comment (tzujenchanmbd) [reply docs/realtime/index.ja.md:46]: Commit pushed https://github.com/MobilityData/gtfs.org/pull/211/commits/17a462d25d141bc12108e24f84569e395442d143

File: docs/realtime/changes/index.ja.md (hunk 2 above)
Comment (tzujenchanmbd) [reply docs/realtime/changes/index.ja.md]: Commit pushed https://github.com/MobilityData/gtfs.org/pull/211/commits/ac6953945b72031d48f788eb58503ca7f8ab9b02

**Review by tzujenchanmbd — APPROVED** [review]

**Comment by isabelle-dr** [comment]
@bdferris-v2 this PR contains the fixes to the Japanese translations if you'd like to review them.

**Comment by bdferris-v2** [comment]
Thanks @isabelle-dr .  I've passed this onto my colleagues and they might make some comments on the PR directly.  I'll let you know when we are done reviewing.

**Review by fredericsimard — COMMENTED** [review]
@tzujenchanmbd About this [commit](https://github.com/MobilityData/gtfs.org/pull/211/commits/deb29cb57857a4c5734e9a368208273ff4c1e465), the `"!!! note """` should not be removed, this is MKDocs code to put the paragraph that follows inside a notes box like in all other languages, see the blue box on this page : https://gtfs.org/realtime/changes/

**Comment by tzujenchanmbd** [comment]
@fredericsimard Have added this back, thanks!

**Pull request merged**